src/metrics-modeling/Dockerfile

```
@@ -0,0 +1,41 @@
+# MetricBot — deetsMeter prediction pipeline (internal service).
+# Design: docs/metrics-modeling/metrics-microservice-deetsmeter.md
+#
+# Build from the REPO ROOT so the build context matches the other
+# services' pipelines:
+#   docker build -f src/metrics-modeling/Dockerfile -t metricbot:local .
+
+FROM python:3.12-slim-bookworm
+
+# psql is the extraction tool (deliberate: no Python DB driver — see
+# metricbot/extract.py). curl is for container-level health checks.
+RUN apt-get update \
```

Hadolint (2.15.1)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)